recipes/protozfits/meta.yaml

```diff
@@ -0,0 +1,74 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "protozfits" %}
+{% set version = "2.0.1.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a3864c6097196208fca15b17eb49241675b3811bd6d1ffec206ce0e3bba8f89f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+  skip: true  # [py<36]
+  missing_dso_whitelist:
+    - "$RPATH/*"
```

Comment:
This shouldn't be needed:
```suggestion
```

Reply:
The build fails without it. The cmake build adds the linked libraries to RPATH and some build checking script complains about not being able to find `$RPATH/libprotobuf.so`

Reply:
This is needed for people installing from source with pip. If this should be deactivated in the conda forge build, what would be a nice solution?

Reply:
Add `libprotobuf` to the `host` dependencies and remove it from `run`

Reply:
Conda-build will then use this to automatically put an appropriate version constraint in `run`.

https://github.com/conda-forge/libprotobuf-feedstock/blob/f1471e3b75f22294942a4693d20db3d14fb3684f/recipe/meta.yaml#L47-L51

Reply:
thanks!